service/controller/v22/resource/migration/resource.go

```diff
@@ -1,145 +0,0 @@
-// Package migration provides an operatorkit resource that migrates awsconfig CRs
-// to reference the default credential secret if they do not already.
-// It can be safely removed once all awsconfig CRs reference a credential secret.
-package migration
```

Comment:
Considering that all resource versions we still reconcile contain the migration resource, all clusters should have the secret reference now. The resource should no longer be necessary.

Reply:
I used migration resource for cluster-autoscaler. Please don't delete this yet.

https://github.com/giantswarm/aws-operator/pull/1338/files#diff-4d2138f74f0509116827828fccee873e

Reply:
Ack. Is this resource a thing that should live longer? I think we should keep track of this a little bit more. Can you update the comment in there so we know what the resource is used for and when we can remove it?